app/internal/rest.py

```diff
@@ -57,3 +58,14 @@ def handler(generic):
         response_body = {generic: request.json}
 
     return response_body, status_code
+
```

Comment:
Covered in tests `tests/app/internal/test_rest.py`

Reply:
We need to resolve the CodeQL issue here. @k-macmillan maybe dismiss as false positive?

Reply:
I'm not sure - This internal endpoint does log the header from the incoming request. 

@k-macmillan 
<img width="767" alt="Screenshot 2025-02-06 at 10 30 28 PM" src="https://github.com/user-attachments/assets/d57cedef-5713-4ba4-9e3f-1c7f795d0b81" />

Reply:
My gut tells me this is not great.

Reply:
We should never be logging the header since the headers all contain bearer tokens. But I don't think that is what CodeQL is complaining about.

Reply:
[Created ticket for this ](https://github.com/department-of-veterans-affairs/notification-api/issues/2304)

Reply:
Please put the ticket number in a comment near this, so the next time it flags we won't create a new ticket. AC of the new ticket should include removal of the comment. You can fill out the ticket details after requesting re-review of this PR.

Reply:
Updated and[ recreated the ticket in the team repo](https://github.com/orgs/department-of-veterans-affairs/projects/1417/views/1?pane=issue&itemId=97305569&issue=department-of-veterans-affairs%7Cvanotify-team%7C1487)